pkg/search/proxy/controller.go

```diff
@@ -246,8 +245,9 @@ func (ctl *Controller) mergeResourcesByClusters(resourcesByClusters map[string]m
 			klog.Errorf("Failed to get gvk: %v", err)
 			continue
 		}
-		if !helper.IsAPIEnabled(cluster.Status.APIEnablements, gvk.GroupVersion().String(), gvk.Kind) {
-			klog.Warningf("Resource %s is not enabled for cluster %s", resource.String(), cluster)
+		apiEnablement := cluster.APIEnablement(gvk)
+		if apiEnablement == clusterv1alpha1.APIDisabled {
```

Comment:
```suggestion
		if cluster.APIEnablement(gvk) == clusterv1alpha1.APIDisabled {
```
Why need an extra variable here?